### DISCUSSION TITLE
Add Sensor for Storage (ZFS/SMB/PBS etc.)

### DISCUSSION THREAD

**Discussion by MosizZ** [dougiteixeira/proxmoxve#199] [Feature request]
The integration is super helpful. I would be very happy if sensors for (ZFS/SMB/PBS etc.) fill levels were included in a future version. Currently only the local storage of the Proxmox host can be read.

**Comment by Serhioromano** [comment]
Duplicate of https://github.com/dougiteixeira/proxmoxve/discussions/114
